### DISCUSSION TITLE
FreeBSD 13 has failing Prisma Build

### DISCUSSION THREAD

**Discussion by mkastner** [keystonejs/keystone#8196] [Questions] [answered]
What a blow! After three days of reading docs and configuring keystone js, I had to find out that Prisma doesn't build on FreeBSD 13. Would have been nice to get Keystone running on FreeBSD, though.

Any help with this?

**Comment by dcousens** [comment, marked as answer]
@mkastner what failed? Is the Prisma team aware of the problem?

**Reply by mkastner** [reply]
@dcousens It looks like for the Prisma developers not providing native FreeBSD builds is not considered a problem. (Yeah, I know, I should have used linux/Docker. Would have been easier.)

Keystone's ```yarn build``` on FreeBSD production doesn't work out of the box. I had to compile from the sources.

I finally found this: [https://github.com/prisma/prisma/issues/3091](https://github.com/prisma/prisma/issues/3091) which solved _that_ problem.
